frontend/components/MapDisplay.vue

```diff
@@ -1,16 +1,11 @@
 <template>
-  <div>
-    <div
-      v-if="loading"
-      class="flex justify-center items-center w-full h-full bg-gray-300 text-gray-400"
-    >
-      <location-marker-icon
-        class="h-36 w-36 animate-pulse"
-        aria-hidden="true"
-      />
-    </div>
-    <div id="map" class="w-full h-full"></div>
+  <div
+    v-if="loading"
+    class="flex justify-center items-center w-full h-full bg-gray-300 text-gray-400"
+  >
+    <location-marker-icon class="h-36 w-36 animate-pulse" aria-hidden="true" />
```

Comment:
How to provoke this situation? I was refreshing the page and I don't see this marker.

Reply:
Already resolved, now it will appear every time, when map is loading